src/components/home/Post.tsx

```diff
@@ -43,14 +43,26 @@ export const Post: React.FC<PostProps> = React.memo(({post, style}) => {
 
   const {name, message, photo} = post;
 
-  const wrapperRef = useRef<HTMLDivElement>(null);
-  const nameRef = useRef<HTMLDivElement>(null);
-  const messageRef = useRef<HTMLDivElement>(null);
+  /**
+   * `wrapperWidth`, width of the main wrapper component – used for when making sure the entire post
+   * is a square or for making the image a square.  
+   * `nameHeight`, height of the component that displays the name – used for computing
+   * the available space left when constraining the post to a square.  
+   * `messageHeight`, height of the component that displays the message – used for computing the 
+   * available space for displaying the image if the message is present.
+   * 
+   * The refs refers to its corresponding components.
+   */
   const [wrapperWidth, setWrapperWidth] = useState<number>(0);
   const [nameHeight, setNameHeight] = useState<number>(0);
   const [messageHeight, setMessageHeight] = useState<number>(0);
```

Comment:
This is good; another way to do it is
```
/** `wrapperWidth`, width of the main wrapper component blha blah  */
const [wrapperWidth, setWrapperWidth] = useState<number>(0);
/** `nameHeight` blah blah */
const [nameHeight, setNameHeight] = useState<number>(0);
```

It doesn't really matter though since I don't think either way will give variables the doc.

Reply:
It's easier to read when all the comments are together.

Reply:
(I might do it the second way :wink:)

Reply:
I like referring to them right above the related variables, but we can certainly shift to that.

Please catch me if I do it the other way though since I'm used to that haha